tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -350,7 +350,7 @@ install_image() {
 	os_name="$(get_from_kata_deps ".assets.image.architecture.${ARCH}.name")"
 	os_version="$(get_from_kata_deps ".assets.image.architecture.${ARCH}.version")"
 	if [ -n "${variant}" ]; then
-		image_type+="-${variant}"
+		image_type+="${variant}-${image_type}"
```

Comment:
We cache based on the build target, which is `rootfs-nvidia-gpu-image` in this case, which is why this is wrong, however the mariner and confidential targets are `rootfs-image-mariner` and `rootfs-image-confidential`, so shouldn't we update the nvidia build target to be `rootfs-initrd-nvidia-gpu-*` otherwise we will break other caches?

Reply:
We do need to do that, and I did in the current version of the PR.